src/atoms/Tooltip/index.js

```diff
@@ -46,12 +48,13 @@ class Tooltip extends React.Component {
           className
         )}
       >
-        <Icon icon='questionMark' className={styles['tooltip']} />
+        { text }
         <span
           className={classnames(
             children && styles['hover-message'],
             left && styles['left'],
             right && styles['right'],
+            hoverMessageClassName && hoverMessageClassName
```

Comment:
If `classnames` pulls out any `undefined` values, could this be just `hoverMessageClassName` without the short-circuiting?

Reply:
ahh, didn't know classnames did that, good call